tests/unit/kgo/job_file.kgo

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash -l
+#
+# ++++ THIS IS A CYLC TASK JOB SCRIPT ++++
+# Suite: farm_noises
+# Task: baa
+# Job log directory: 1/baa/01
+# Job submit method: background
+# Job submit command template: woof
+# Execution time limit: moo
+if [[ $1 == 'noreinvoke' ]]; then
+    shift
+else
+    exec bash -l "$0" noreinvoke "$@"
+fi
+
+CYLC_FAIL_SIGNALS='EXIT ERR TERM XCPU'
+export CYLC_VERSION='8.0a3.dev'
```

Comment:
Need to replace the version with the current version (this is what caused the last breakage).

Reply:
Dun

Reply:
Every time the cylc version changes this test will break.